### PR TITLE
log4j2 liquibase fix

### PR DESCRIPTION
**Pull request by mallireddy15**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -211,10 +218,18 @@
                 <version>2.13.3</version>
             </dependency>
 
+<!--            <dependency>-->
```
Comment (pintea-eugen) [inline pom.xml:221]: This dependency can be removed, not just commented out.

File: pom.xml [hunk 2]
```diff
@@ -211,10 +218,18 @@
...
+<!--            </dependency>-->
+
+            <!-- log4j2 dependency for liquibase -->
             <dependency>
-                <groupId>org.slf4j</groupId>
-                <artifactId>slf4j-log4j12</artifactId>
-                <version>1.7.16</version>
+                <groupId>org.apache.logging.log4j</groupId>
+                <artifactId>log4j-slf4j-impl</artifactId>
+                <version>2.14.1</version>
```
Comment (pintea-eugen) [inline pom.xml:232]: I would suggest to stick with the 2.13.3 version, for consistency reasons.

File: pom.xml (hunk 1 above)
Comment (mallireddy15) [reply pom.xml]: Removed from pom.xml

File: pom.xml (hunk 2 above)
Comment (mallireddy15) [reply pom.xml]: Changed the version as suggested

**Review by pintea-eugen — APPROVED** [review]
Looks good! 👍

**Pull request merged**